src/content/docs/en/guides/deploy/zeabur.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Deploy your Astro Site to Zeabur
+description: How to deploy your Astro site to the web on Zeabur.
+type: deploy
+i18nReady: true
+---
+[Zeabur](https://zeabur.com) offers hosting for full-stack web applications. Any Astro site can be hosted on Zeabur as an SSR or static site.
+
+This guide includes instructions for deploying to Zeabur through the website UI.
+
+## Project Configuration
+
+Your Astro project can be deployed to Zeabur in two different ways: as a static site, or a server-rendered site.
+
+### Static Site
+
+Your Astro project is a static site by default. You don’t need any extra configuration to deploy a static Astro site to Zeabur. 
+
+### Server-Rendered Site
+
+To deploy with SSR support, make sure you first set up the @astrojs/node adapter. Then, follow the following steps to configure your Astro site for SSR:
+
+1. Add the following to your `astro.config.mjs` file:
+
+    ```js
+    import node from '@astrojs/node';
+    
+    export default defineConfig({
+      output: 'server',
+      adapter: node({
+        mode: "standalone"
+      })
+    })
+    ```
+
+2. Modify the start command in your `package.json` file:
+
+    ```json
+    {
+      "scripts": {
+        "start": "node dist/server/entry.mjs"
+      }
+    }
+    ```
+
+## How to deploy
+
+You can deploy to Zeabur through the website UI.
+
+### Website UI Deployment
```

Comment:
I would opt to just remove both sentences here, since they don't add any extra value
```suggestion
```

Reply:
That's because Zeabur's CLI is still in a beta version, I'll remove these and come back with another pr when the CLI is ready!